__tests__/app.test.js

```diff
@@ -66,3 +66,30 @@ describe("getArticleById", () => {
       });
   });
 });
+describe("updateVotes", () => {
+  test("status: 200 and responds with the updated votes in the article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .expect(200)
+      .send({ inc_votes: 1 })
+      .then(({ body: { article } }) => {
+        expect(article.votes).toBe(101);
+      });
+  });
+  test("status:404 sends error message when given a valid but non-existent address", () => {
+    return request(app)
+      .patch("/api/article/1")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Invalid address");
+      });
+  });
+  test("status:400 sends error message when given an invalid address", () => {
+    return request(app)
+      .patch("/api/articles/vote")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
+      });
+  });
+});
```

Comment:
missing some tests. What you try and increment by a random string? eg `{inc_votes:'banana'}`

Reply:
valid but non-existent article_id?